content/blog/1.0-announcement/index.mdx

```diff
@@ -0,0 +1,73 @@
+
+  
+
+Declarative is usually the easiest to understand. Not just infrastructure-as-code but all configuration data that is needed to run a system including the application layer. While this doesn’t generally include persistent application data such as database contents it generally includes credentials and configuration for data recovery or data access.
+
+  
+
+Versioned and Immutable is often understood as “use git” but there is more to it than that. If that desired state describes using a “latest” tag for example, it is no longer versioned because there is no way to rollback changes in the desired state. Likewise, many version control systems can be used in GitOps as long as they meet those two basic requirements and teams use them in a conformant manner.
+
+  
+
+Pulled Automatically means we have to have software agents constantly observing the desired state. While triggers may exist to speed up that observation, a GitOps system shouldn’t exclusively rely on them. Software agents need to be smart enough to constantly be aware of what the desired state should be. The verb “pull” here is used very clearly in contrast to the way traditional CD pipelines function based on triggers that simply push.
+
+  
+
+Finally, Continuously Reconciling brings it all together. The GitOps software agents have to be aware of the actual state of a system under management and attempt to apply the desired state. Being constantly aware of both the actual state and desired state means we can detect anytime they are out of alignment whether by changes to the desired state (normal operation) or changes to the actual state (drift), a GitOps software agent should detect this attempt to apply the desired state.
+
+  
+
+When teams implement these principles and achieve GitOps they deploy more often, have fewer regressions, and are more competitive. They bring together decades of experience delivering software to create a standard that is incredibly powerful and accessible and rapidly being adopted by a growing number of open source tools and vendors.
+
+## What’s next
+
+With the publication of this standard we’re announcing two new initiatives to help more and more teams adopt these principles. First, “GitOps Blueprints” will be a place where anyone from the community can bring their stacks and share how they’ve implemented GitOps using various tools. Want to know how others have implemented GitOps using your favorite cloud, SCM, and orchestrator? GitOps Blueprints is a place where you’ll be able to see examples from people implementing GitOps in practice.
+
+  
+
+Second, “GitOps Whitepapers” is a place to explore ideas around implementing the GitOps principles. For example, what if your SCM system becomes unavailable? When is it ok to “break glass” and start making manual changes? GitOps Whitepapers are meant to be a space where the community can share these ideas and document them.
```

Comment:
```suggestion
Second, “GitOps Green papers” is a place to explore ideas around implementing the GitOps principles. For example, what if your SCM system becomes unavailable? When is it ok to “break glass” and start making manual changes? GitOps Green papers are meant to be a space where the community can share these ideas and document them.
```

The description sounds more like Green papers than White papers.

According to https://en.wikipedia.org/wiki/White_paper
> By contrast, green papers, which are issued much more frequently, are more open-ended. Also known as consultation documents, green papers may merely propose a strategy to implement in the details of other legislation, or they may set out proposals on which the government wishes to obtain public views and opinion.

If App Delivery TAG GWG is a vendor in business-to-business marketing, then Whitepaper would make more sense.  However, App Delivery TAG GWG behaves more like a regulatory government body in the context of conformance.  Hence the suggestion to use Green papers.

Reply:
Oh interesting, I've never heard of green papers before!

> White papers are a way the government can present policy preferences before it introduces legislation. Publishing a white paper tests public opinion on controversial policy issues and helps the government gauge its probable impact.

This sounds pretty close to what we're going for with these. Places people can bring policy preferences. I don't know though, what do you think @scottrigby @murillodigital ?

Reply:
@todaywasawesome @scottrigby @murillodigital One reason I suggest green papers is because @todaywasawesome wrote `"to explore ideas"` which aligns with how green papers are more open-ended for public consultation.

For example:

• [**Know your green paper on marriages in South Africa**](http://www.dha.gov.za/index.php/notices/1449-know-your-green-paper-on-marriages-in-south-africa) at http://www.dha.gov.za/index.php/notices/1449-know-your-green-paper-on-marriages-in-south-africa

* [**May 2021: Green paper on marriages in South Africa**](https://www.gov.za/sites/default/files/gcis_document/202105/44529gon398.pdf) at https://www.gov.za/sites/default/files/gcis_document/202105/44529gon398.pdf

• [**Green Papers are documents published by the European Commission to stimulate discussion on given topics at European level**](https://eur-lex.europa.eu/summary/glossary/green_paper.html) at https://eur-lex.europa.eu/summary/glossary/green_paper.html

* [**January 2021: Green Paper on Ageing: Fostering solidarity and responsibility between generations**](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021DC0050) at https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021DC0050

WDYT?  Thank you!

Reply:
@todaywasawesome @scottrigby @murillodigital One reason I suggest green papers is because @todaywasawesome wrote `"to explore ideas"` which aligns with how green papers are more open-ended for public consultation.

For example:

• [**Know your green paper on marriages in South Africa**](http://www.dha.gov.za/index.php/notices/1449-know-your-green-paper-on-marriages-in-south-africa) at http://www.dha.gov.za/index.php/notices/1449-know-your-green-paper-on-marriages-in-south-africa

* [**May 2021: Green paper on marriages in South Africa**](https://www.gov.za/sites/default/files/gcis_document/202105/44529gon398.pdf) at https://www.gov.za/sites/default/files/gcis_document/202105/44529gon398.pdf

• [**Green Papers are documents published by the European Commission to stimulate discussion on given topics at European level**](https://eur-lex.europa.eu/summary/glossary/green_paper.html) at https://eur-lex.europa.eu/summary/glossary/green_paper.html

* [**January 2021: Green Paper on Ageing: Fostering solidarity and responsibility between generations**](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021DC0050) at https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021DC0050

WDYT?  Thank you!

Reply:
I personally think this is super interesting. Let's just discuss as a larger group. Can move this idea to a GitHub discussion, and also add to the next meeting agenda, with a goal of getting more feedback into that discussion. that way this blog won't be held up on those details. Does that sound good?